### ISSUE TITLE
gplvm gnuplot inserts invalid lines in files 

### ISSUE THREAD

**Issue by wadaniel** [SheffieldML/GPc#6]
./gplvm gnuplot oil100.model oil100

adds following lines at the beginning of the file that cannot be parsed by gnuplot

version=0.200000
baseType=matrix
type=doubleMatrix
numRows=1000
numCols=3

in files *_variance_matrix.dat and *_latent_data*.dat

I need to remove them otherwise gnuplot cannot parse the files

**Comment by lawrennd** [comment]
Yes, I think that's an update to one part of the API that broke gnuplot compatibility.

It's not something I'll have time to fix at the moment, but feel free to take a look at the code if you have time.
